src/accelerate/accelerator.py

```diff
@@ -3053,3 +3057,16 @@ def skip_first_batches(self, dataloader, num_batches: int = 0):
     def __deepcopy__(self, memo):
         logger.info("Deep copying the `Accelerator` object, note that this will point to the same original object.")
         return self
+
+    def verify_device_map(self, model: torch.nn.Module) -> bool:
```

Comment:
Note that in Accelerate like in Transformers (which is need to be better documented since I know it's not canonical Python), things that are not in one of the inits (main or utils basically) are considered private even if they don't begin with a `_`.

Reply:
@BenjaminBossan with this also is selectively showing certain functions in our documentation as well (this being more along the route of everything buried in `utils/`)